core/arch/arm/plat-imx/conf.mk

```diff
@@ -460,5 +460,30 @@ CFG_IMX_CAAM ?= y
 endif
 endif
 
+# This is only here to force an informative print showing that the device runs
+# with an insecure configuration. Configuring the CSU correctly for end
+# products is a necessary step that the device manufacturer need to do to be
+# able to end up with a secure end product.
+#
+# It should be noted that flipping the flag to "n" doesn't change any CSU
+# configuration at all. I.e., this flag doesn't affect any CSU settings at all.
+# It only enables or disables an informative print serving as a reminder to the
+# device manufacturer that they need to make changes to the CSU.
+#
+# The device manufacturer should:
+#  1. Consult the NXP security reference manual and configure the CSU
+#     accordingly. NXP shares the security reference manual with their partners
+#     under a NDA. I.e., partners will need to sign a NDA with NXP to get hold
+#     of the security reference manual. Because of this the OP-TEE project
+#     cannot and are not allowed to share the security reference manual.
+#  2. Once the CSU configuration has been completed and verified that it is
+#     working, this flag should be flipped to "n".
+#
+# For more information about some of the implications when not setting this
+# correctly can be found at:
+#   - https://github.com/OP-TEE/optee_os/security/advisories/GHSA-6q85-3ph3-rm47
+#   - https://github.com/OP-TEE/optee_os/security/advisories/GHSA-4pqr-q8rf-8464
```

Comment:
This URL is currently only working for OP-TEE admins. But once we've moved it to "public" it will work for anyone.